environment.yml

```diff
@@ -40,4 +40,7 @@ dependencies:
   - nvidia-ml-py3
   - nteract-scrapbook
   - azureml-sdk[notebooks,contrib]>=1.0.30
+  - decord==0.3.5
+  - einops==0.1.0
+  - numpy==1.17.4 # https://github.com/pytorch/vision/issues/1700#issuecomment-569327090
```

Comment:
- Would be better to install these new libraries in the conda, rather than pip, section to avoid dependency problems. I had a look though and seems that both decord and einops are pip only.
- Young made an update to the cocoapi which hopefully means we don't need to revert numpy to a specific version. I am afraid this would give us headaches in the future.

Reply:
+1